ExampleMod/Content/Tiles/ExampleSign.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Terraria.DataStructures;
+using Terraria.ModLoader;
+using Terraria.ObjectData;
+using Terraria;
+using Microsoft.Xna.Framework;
+using Terraria.Enums;
+using Terraria.Localization;
+using Terraria.ID;
+using Terraria.GameContent.ObjectInteractions;
+using Terraria.Chat;
+
+namespace ExampleMod.Content.Tiles
+{
+	public class ExampleSign : ModTile
+	{
+		public override void SetStaticDefaults()
+		{
+			// Note: The max amount of signs in a world is 1000 (the size of the tileSign array).
+			Main.tileSign[Type] = true;
```

Comment:
Are you saying that if there are 1000 tiles loaded, and the last one happens to be ExampleSign, it will IOOB?

Reply:
This comment has an error, thank you for catching it! The comment is mistakenly referencing Main.tileSign[] when it should be referencing Main.sign[]. 
I've updated the comment to be more clear, and moved it to the PlaceInWorld section where it makes more sense.

Updated with https://github.com/tModLoader/tModLoader/pull/4269/commits/b3429f2eb320a9b6257ae914a3d3010ec38a88a3